### DISCUSSION TITLE
Laravel Livewire [Data] Dump on Ray

### DISCUSSION THREAD

**Discussion by spooksie** [spatie/ray#772] [Ideas]
Whilst I love the Chrome Addon for Livewire. It would be great to see a dump of all the public variables showing up on Ray.

If this can be done already, please let me know. 

Many thanks for an awesome app.

**Comment by patinthehat** [comment]
@spooksie I'm working on a livewire  integration now, and I hope to release it in the next month or so. I'll update here when it's public!
